daemons/aws_elasticsearch_delete_index.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+from urllib.parse import urlparse, urlencode, parse_qs
+
+import boto3
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
+from botocore.vendored import requests
+from elasticsearch import Elasticsearch, RequestsHttpConnection
+
+logging.basicConfig(level=logging.WARNING)
+log = logging.getLogger(__name__)
+
+#
+# Delete an index from an AWS Elasticsearch domain.
+#
+# arguments:
+#   -h, --help            show this help message and exit
+#   -d DOMAINNAME, --domainname DOMAINNAME
+#                         Elasticsearch domain name
+#   -i INDEX, --index INDEX
+#                         Name of index to delete or "_all"
+#
```

Comment:
This docstring is redundant given the one automatically generated by argparse. If you want to keep it, I suggest moving it up to the top of the file and using triple-quote (""") comment block, and then using this when you declare the argparse parser:
`parser = argparse.ArgumentParser(description=__doc__, formatter_class=argparse.RawTextHelpFormatter)`